Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -18943,7 +18949,6 @@
 			isa = PBXTargetDependency;
 			platformFilters = (
 				ios,
-				xros,
```

Comment:
These changes look unrelated.  Let's undo them even though Xcode probably did this automatically.

Reply:
Whoops, will remove these.